test/Experimental/MatrixGroups-test.jl

```diff
@@ -0,0 +1,9 @@
+@testset "GAP matrix groups" begin
+
+    m = matrix(ZZ, [0 1 ; -1 0])
+    gapM = wrap_for_gap(m)
+    G = GAP.Globals.Group(gapM)
+    GAP.Globals.SetNiceMorphismForJuliaMatrixRepGroup(G)
+    @test GAP.Globals.Size(G) == 4
```

Comment:
This is a good first proof of concept. Next interesting thing would be to combine this with `isomorphic_group_over_finite_field`: this function is for groups over OSCAR number fields: it determines if they are finite and if so produces an isomorphic group over a finite field. Right now this function (resp. `_isomorphic_group_over_finite_field`) takes a bunch of OSCAR matrices, say `nfmats`, and returns several objects, among them a group over finite fields isomorphic to the group that would have been generated by `nfmats`.

So right now using these groups is a bit cumbersome. Instead, one should be allowed to just write `MatrixGroup(nfmats)` (or in fact just `group(nfmats)`!) and then the "right thing" happens: create a GAP group whose GAP generators are wrappers of these Julia matrices, and which has a nice monomorphism -- but into a GAP group over finite fields, using a homomorphism coming from code adapted from `_ _isomorphic_group_over_finite_field`.